app/src/pages/NotFound.vue

```diff
@@ -18,16 +18,17 @@
                         </ul>
                     </div>
                     <div class="u--inline">
-                        <div v-if="$store.getters.isAuthenticated" class="nav_menu--container">
+                        <div v-if="$store.getters['auth/isAuthenticated']" class="nav_menu--container">
                             <a class="u--default-size nav_menu--handler" style="color:#fff; font-size:1.2rem !important;">
                                 <i class="material-icons" style="vertical-align: middle;">perm_identity</i>
+                                {{ $store.getters['auth/displayName'] }}
                             </a>
                             <div class="nav_menu--siblings nav_menu--sibheader">
                                 <span class="nav_menu--siblings-lists" @click="$store.dispatch('logout')"><a>Logout</a></span>
                             </div>
                         </div>
                         <div v-else>
-                            <a class="btn btn--tertiary btn--noradius">Login/Register</a>
+                            <a class="btn btn--tertiary btn--noradius" href="/secure">Login/Register</a>
```

Comment:
Should we match the styling on this button to the Logout one? It seems to be showing the blue underline instead of the usual menu highlighting

Reply:
> Should we match the styling on this button to the Logout one? It seems to be showing the blue underline instead of the usual menu highlighting

<img width="1791" alt="Screenshot 2023-02-21 at 4 25 22 PM" src="https://user-images.githubusercontent.com/94853475/220461991-b712bf4b-a99f-4e3f-9d49-1650e2a49449.png">

It looks okay for me for some reason

Reply:
Oops, I commented on the wrong one